[I 241120 11:34:33 command:168] Visit me at http://0.0.0.0:5555
[I 241120 11:34:33 command:176] Broker: redis://localhost:6379/0
[I 241120 11:34:33 command:177] Registered tasks: 
    ['celery.accumulate',
     'celery.backend_cleanup',
     'celery.chain',
     'celery.chord',
     'celery.chord_unlock',
     'celery.chunks',
     'celery.group',
     'celery.map',
     'celery.starmap',
     'celery_worker.task.process_task',
     'celery_worker.task.run_user_automation']
[I 241120 11:34:33 mixins:228] Connected to redis://localhost:6379/0
[W 241120 11:34:34 inspector:44] Inspect method revoked failed
[W 241120 11:34:34 inspector:44] Inspect method conf failed
[W 241120 11:34:34 inspector:44] Inspect method reserved failed
[W 241120 11:34:34 inspector:44] Inspect method scheduled failed
[W 241120 11:34:34 inspector:44] Inspect method active_queues failed
[W 241120 11:34:34 inspector:44] Inspect method stats failed
[W 241120 11:34:34 inspector:44] Inspect method registered failed
[W 241120 11:34:34 inspector:44] Inspect method active failed
[E 241120 12:03:52 events:191] Failed to capture events: 'Connection closed by server.', trying again in 1 seconds.
[E 241120 12:03:59 events:191] Failed to capture events: 'Error 61 connecting to localhost:6379. Connection refused.', trying again in 2 seconds.
[E 241120 12:04:07 events:191] Failed to capture events: 'Error 61 connecting to localhost:6379. Connection refused.', trying again in 4 seconds.
[E 241120 12:04:17 events:191] Failed to capture events: 'Error 61 connecting to localhost:6379. Connection refused.', trying again in 8 seconds.
[E 241120 12:04:31 events:191] Failed to capture events: 'Error 61 connecting to localhost:6379. Connection refused.', trying again in 16 seconds.
[E 241120 12:04:53 events:191] Failed to capture events: 'Error 61 connecting to localhost:6379. Connection refused.', trying again in 32 seconds.
[E 241120 12:05:31 events:191] Failed to capture events: 'Error 61 connecting to localhost:6379. Connection refused.', trying again in 64 seconds.
[E 241120 12:06:41 events:191] Failed to capture events: 'Error 61 connecting to localhost:6379. Connection refused.', trying again in 128 seconds.
[E 241120 12:08:55 events:191] Failed to capture events: 'Error 61 connecting to localhost:6379. Connection refused.', trying again in 256 seconds.
[E 241120 12:13:17 events:191] Failed to capture events: 'Error 61 connecting to localhost:6379. Connection refused.', trying again in 512 seconds.
[E 241120 12:21:55 events:191] Failed to capture events: 'Error 61 connecting to localhost:6379. Connection refused.', trying again in 1024 seconds.
